sorting/inversions.cpp

```diff
@@ -0,0 +1,145 @@
+/**
+ * @file
+ * @brief [https://www.hackerrank.com/challenges/ctci-merge-sort/problem]
```

Comment:
Is this merge sort? If so, brief description should be like the following.
```suggestion
 * @brief [Merge sort](https://www.hackerrank.com/challenges/ctci-merge-sort/problem) algorithm
```

Reply:
Inversion count is an application of the merge sort algorithm, that's why the link appears to be of merge sort. The link explains the inversion concept. I have made the changes suggested by you